src/clients/patreon/patreonAgent.ts

```diff
@@ -8,27 +8,23 @@ export interface PatreonUserTokens {
 
 export interface PatronFullDetails {
   patreonUserId: string;
-  isMember: Boolean;
-  // Below 4 attributes will be null if isMember is false
-  lastChargeDate: Date;
-  lastChargeStatus: string;
-  nextChargeDate: Date;
+  isPaidPatron: Boolean;
```

Comment:
Let's not have the validity of this object depend on a specific random thing that the users of this interface need to know.